### PR TITLE
Add brazil servers

### PR DESCRIPTION
**Pull request by rafaelbbraz**
Just add Brazil servers.
I need to add the repo **l10n_br** to ofn git account to. but for whille I still using ofb git account to host this repo.

### CODE REVIEW

File: inventory/group_vars/br.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+checkout_zone: Brazil
+country_code: BR
+currency: BRL
+
+locale: pt_BR # Default locale for OFN instance
+available_locales: pt_BR # Available locales for OFN instance
+language: pt_BR.UTF-8 # Default language for the server
+language_packages: # Language packages for the server
+  - language-pack-en-base
+l10n_repo: https://github.com/openfoodbrazil/l10n_br.git
```
Comment (luisramos0) [inline inventory/group_vars/br.yml:12]: I have imported l10n_br to ofn :tada: 
you can now switch this line:
```suggestion
l10n_repo: https://github.com/openfoodfoundation/l10n_br.git
```

File: inventory/hosts [hunk 2]
```diff
@@ -169,6 +183,7 @@ us-prod
 europe
 america
 au-prod
+br-prod
```
Comment (luisramos0) [inline inventory/hosts:186]: this group is for servers maintained by the global team, so BR, for now, should not be here.
you can delete this line.

File: inventory/hosts [hunk 3]
```diff
@@ -178,6 +193,7 @@ au-staging
 uk-staging
 fr-staging
 es-staging
+br-staging
```
Comment (luisramos0) [inline inventory/hosts:196]: same here, this group is for servers maintained by the global team.
this line can be deleted.

File: inventory/hosts [hunk 4]
```diff
@@ -195,3 +211,4 @@ fr
 de
 be
 za
+br
```
Comment (luisramos0) [inline inventory/hosts:214]: this is the full list of servers, br should be here :+1:

File: inventory/group_vars/br.yml (hunk 1 above)
Comment (Matt-Yorkley) [reply inventory/group_vars/br.yml:12]: This change should be good now that https://github.com/openfoodfoundation/l10n_br.git has been added :+1:

File: inventory/hosts [hunk 5]
```diff
@@ -157,6 +170,7 @@ de-prod
 [america:children]
 ca-prod
 us-prod
+br-prod
```
Comment (Matt-Yorkley) [inline inventory/hosts:173]: This line needs to be removed :fire:

File: inventory/hosts (hunk 2 above)
Comment (Matt-Yorkley) [reply inventory/hosts:182]: This line needs to be removed :fire:

File: inventory/hosts (hunk 3 above)
Comment (Matt-Yorkley) [reply inventory/hosts:192]: This line needs to be removed :fire:

**Comment by Matt-Yorkley** [comment]
A quick explanation: some instances/countries contribute to the financing of OFN, and are managed by the (centrally paid) sysadmin team, and some instances are a bit outside of that group or are just starting, and are managed independently. These configs reflect that (not very clearly), and actually define which servers our playbooks will be run against when the core team is performing different operations on production servers.

**Pull request merged**